apps/addons/decorators.py

```diff
@@ -10,6 +10,12 @@
 log = commonware.log.getLogger('mkt.purchase')
 
 
+def owner_or_unlisted_reviewer(request, addon):
+    return (acl.check_unlisted_addons_reviewer(request) or
+            acl.check_addon_ownership(request, addon, admin=False, dev=True,
```

Comment:
We have the check for the "unlisted addons reviewer" (which contains the site admins). What we don't want, is "admins" (in the sense it's used in this helper: anyone with "Addons:Edit") to have access to those.